test/Microsoft.AspNet.FileProviders.Embedded.Tests/project.json

```diff
@@ -2,17 +2,23 @@
     "dependencies": {
         "Microsoft.AspNet.FileProviders.Embedded": "1.0.0-*",
         "Microsoft.AspNet.Testing": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
+        "xunit": "2.1.0"
     },
+    "testRunner": "xunit",
     "commands": {
         "test": "xunit.runner.aspnet"
```

Comment:
We still use the dnx runner for coreclr. The xunit runner for dotnet doesn't work cross platform